mock-core-configs/etc/mock/templates/oraclelinux-9.tpl

```diff
@@ -0,0 +1,68 @@
+config_opts['chroot_setup_cmd'] = 'install tar redhat-rpm-config redhat-release oraclelinux-release which xz sed make bzip2 gzip coreutils unzip shadow-utils diffutils cpio bash gawk rpm-build info patch util-linux findutils grep glibc-minimal-langpack'
+config_opts['dist'] = 'el9'  # only useful for --resultdir variable subst
+config_opts['releasever'] = '9'
+config_opts['package_manager'] = 'dnf'
+config_opts['extra_chroot_dirs'] = [ '/run/lock', ]
+config_opts['bootstrap_image'] = 'docker.io/library/oraclelinux:9'
```

Comment:
According to https://github.com/containers/shortnames/commit/d30ea407b747da6159936b99a444c5ae31bb6fc8 the official image for Oracle Linux is `container-registry.oracle.com/os/oraclelinux`, so we should use that instead for the bootstrap image.

Reply:
thank you, fixed!